Tests/Functional/Services/DeeplServiceTest.php

```diff
@@ -95,6 +152,18 @@ public function checkSupportedTargetLanguages(): void
         static::assertNull($deeplService->detectTargetLanguage('BS'));
     }
 
+    /**
+     * @test
+     */
+    public function checkIsTargetLanguageSupported(): void
+    {
+        /** @var DeeplService $deeplService */
+        $deeplService = $this->get(DeeplService::class);
+
+        static::assertTrue($deeplService->isTargetLanguageSupported('DE'));
+        static::assertFalse($deeplService->isTargetLanguageSupported('BS'));
```

Comment:
Just a reminder, because I saw it here: We should avoid using a real existing language here, as the tests will fail, if the language gets supported by DeepL and the mock server is updated.